xml/System.Numerics/Vector.xml

```diff
@@ -2127,9 +2137,9 @@ Note that this method returns a <xref:System.Single> instead of an integral type
         <Parameter Name="value" Type="System.Numerics.Vector&lt;System.Double&gt;" Index="0" FrameworkAlternate="net-9.0" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">The vector to convert.</param>
+        <summary>Converts a <see cref="T:System.Numerics.Vector`1" /> to a <see cref="T:System.Numerics.Vector`1" /> using platform specific behavior on overflow.</summary>
```

Comment:
Looks like some type information was lost here, believe we've hit this issue in the past as well.

Reply:
Same comment applies to several other methods here